### ISSUE TITLE
Separating scopes by groups is not working.

### ISSUE THREAD

**Issue by oni7uka** [activeadmin/activeadmin#5497]
I've seen in the doc [https://activeadmin.info/3-index-pages.html](https://activeadmin.info/3-index-pages.html) at `Index Scopes` section that I can separate my scopes in groups.
So I tried adding:
```
  scope :all
  scope :active, group: :status
  scope :inactive, group: :status
  
  Setting.distinct.pluck(:namespace).each do |namespace|
    scope(namespace,group: :namespace){ |scope| scope.where(namespace: namespace) } 
  end 
```
But nothing happen:
![issue](https://user-images.githubusercontent.com/15939343/46252876-c0e7de00-c4a1-11e8-81a2-4d29041fb1f9.png)
The jvccbar is  defined in `namespace` group, so i think it should be like this: ( all ) ( acitve | inactive) ( jvccbar ), Do I missing something ? 
I'm using ruby version `2.5.1`,  rails version `5.2.1`, activeadmin version `1.3.1`

**Comment by deivid-rodriguez** [comment]
Ths is because this feature has not yet been released, so there's indeed a problem here: discrepancy because the published docs in the website and the released features. We need to figure out how to tackle that, I'll open a new ticket.

Regarding the [groups feature](https://github.com/activeadmin/activeadmin/pull/5359), you can opt in for now by pointing your app against master.

**Comment by varyonic** [comment]
Has been released in 1.4.

**Comment by deivid-rodriguez** [comment]
Indeed, thanks!
